# HG changeset patch
# User leonidr
# Date 1382726240 -3600
#      Fri Oct 25 19:37:20 2013 +0100
# Node ID e7ed5dad88eefc49130ae02bcbf329d72f18f12e
# Parent  47c881c5439a4542b19ab85f376e94fc2c0a5a57
8014718: Netbeans IDE begins to throw a lot exceptions since 7u25 b10
Summary: Removed logging from SunToolkit
Reviewed-by: art, omajid

diff -r 47c881c5439a -r e7ed5dad88ee src/share/classes/java/awt/Toolkit.java
--- openjdk/jdk/src/share/classes/java/awt/Toolkit.java	Wed May 01 00:49:21 2013 +0200
+++ openjdk/jdk/src/share/classes/java/awt/Toolkit.java	Fri Oct 25 19:37:20 2013 +0100
@@ -48,8 +48,6 @@
 import java.io.FileInputStream;
 
 import java.util.*;
-import java.util.logging.*;
-
 import java.beans.PropertyChangeListener;
 import java.beans.PropertyChangeSupport;
 import sun.awt.AppContext;
@@ -1922,7 +1920,7 @@
      */
     public abstract boolean isModalExclusionTypeSupported(Dialog.ModalExclusionType modalExclusionType);
 
-    private static final Logger log = Logger.getLogger("java.awt.Toolkit");
+    // 8014736: logging has been removed from Toolkit
 
     private static final int LONG_BITS = 64;
     private int[] calls = new int[LONG_BITS];
@@ -2089,12 +2087,6 @@
         }
 
     synchronized int countAWTEventListeners(long eventMask) {
-        if (log.isLoggable(Level.FINE)) {
-            if (eventMask == 0) {
-                log.log(Level.FINE, "Assertion (eventMask != 0) failed");
-            }
-        }
-
         int ci = 0;
         for (; eventMask != 0; eventMask >>>= 1, ci++) {
         }
diff -r 47c881c5439a -r e7ed5dad88ee src/share/classes/sun/awt/SunToolkit.java
--- openjdk/jdk/src/share/classes/sun/awt/SunToolkit.java	Wed May 01 00:49:21 2013 +0200
+++ openjdk/jdk/src/share/classes/sun/awt/SunToolkit.java	Fri Oct 25 19:37:20 2013 +0100
@@ -58,7 +58,7 @@
     implements WindowClosingSupport, WindowClosingListener,
     ComponentFactory, InputMethodSupport {
 
-    private static final Logger log = Logger.getLogger("sun.awt.SunToolkit");
+    // 8014736: logging has been removed from SunToolkit
 
     /* Load debug settings for native code */
     static {
@@ -554,10 +554,6 @@
         if (event == null) {
             throw new NullPointerException();
         }
-        AppContext eventContext = targetToAppContext(event.getSource());
-        if (eventContext != null && !eventContext.equals(appContext)) {
-            log.fine("Event posted on wrong app context : " + event);
-        }
         PostEventQueue postEventQueue =
             (PostEventQueue)appContext.get(POST_EVENT_QUEUE_KEY);
         if(postEventQueue != null) {
@@ -938,10 +934,6 @@
             //with scale factors x1, x3/4, x2/3, xN, x1/N.
             Image im = i.next();
             if (im == null) {
-                if (log.isLoggable(Level.FINER)) {
-                    log.log(Level.FINER, "SunToolkit.getScaledIconImage: " +
-                            "Skipping the image passed into Java because it's null.");
-                }
                 continue;
             }
             if (im instanceof ToolkitImage) {
@@ -954,10 +946,6 @@
                 iw = im.getWidth(null);
                 ih = im.getHeight(null);
             } catch (Exception e){
-                if (log.isLoggable(Level.FINER)) {
-                    log.log(Level.FINER, "SunToolkit.getScaledIconImage: " +
-                            "Perhaps the image passed into Java is broken. Skipping this icon.");
-                }
                 continue;
             }
             if (iw > 0 && ih > 0) {
@@ -1029,14 +1017,6 @@
         try {
             int x = (width - bestWidth) / 2;
             int y = (height - bestHeight) / 2;
-            if (log.isLoggable(Level.FINER)) {
-                log.log(Level.FINER, "WWindowPeer.getScaledIconData() result : " +
-                        "w : " + width + " h : " + height +
-                        " iW : " + bestImage.getWidth(null) + " iH : " + bestImage.getHeight(null) +
-                        " sim : " + bestSimilarity + " sf : " + bestScaleFactor +
-                        " adjW : " + bestWidth + " adjH : " + bestHeight +
-                        " x : " + x + " y : " + y);
-            }
             g.drawImage(bestImage, x, y, bestWidth, bestHeight, null);
         } finally {
             g.dispose();
@@ -1047,10 +1027,6 @@
     public static DataBufferInt getScaledIconData(java.util.List<Image> imageList, int width, int height) {
         BufferedImage bimage = getScaledIconImage(imageList, width, height);
         if (bimage == null) {
-             if (log.isLoggable(Level.FINER)) {
-                 log.log(Level.FINER, "SunToolkit.getScaledIconData: " +
-                         "Perhaps the image passed into Java is broken. Skipping this icon.");
-             }
             return null;
         }
         Raster raster = bimage.getRaster();
